pl.art.lach.slawek.apps.v2blankbrowser.json

```
@@ -0,0 +1,36 @@
+{
+  "id": "pl.art.lach.slawek.apps.V2BlankBrowser",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.38",
```

```suggestion
  "runtime-version": "41",
```

runtime 3.38 is End-Of-Life